mongodb/trigger/eventlistener/metadata.go

```diff
@@ -0,0 +1,96 @@
+package eventlistener
+
+import (
+	"github.com/project-flogo/core/data/coerce"
+)
+
+// Settings structure
+type Settings struct {
+	Connection string `md:"mongodbConnection,required"`
+}
+
+// HandlerSettings structure
+type HandlerSettings struct {
+	Collection   string `md:"collection"`
+	ListenInsert bool   `md:"listenInsert,required"`
+	ListenUpdate bool   `md:"listenUpdate,required"`
+	ListenRemove bool   `md:"listenRemove,required"`
+}
+
+// Output structure
+type Output struct {
+	Output map[string]interface{} `md:"Output"`
```

Comment:
I think we should stick with "md:"output.""

Reply:
Will change..